src/nrc/setup_configuration/authorization.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
```

Comment:
I see I didn't add these for other newly added files, apparently this was never checked in CI? Do we need these comments at all?

Reply:
I think we only have a check for it in Open Zaak
So it can be definitely removed here (and I'm not sure why we keep it in OZ, maybe we don't have to)